geode-dunit/src/main/java/org/apache/geode/test/dunit/internal/DUnitLauncher.java

```diff
@@ -395,12 +395,18 @@ public static void closeAndCheckForSuspects() {
           + "' - perhaps a rule that is cleaning up before suspect processing has already run.");
     }
 
+    boolean ok = true;
```

Comment:
An alternative is to create a `StringBuilder` here and pass it into `checkSuspectFile()` to collect the suspects. That way the strings can be included in the assertion failure message.

Reply:
Good idea - I'll apply that shortly.